speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/MediatorTests.cpp

```diff
@@ -197,24 +197,23 @@ END_TEST_COMBINATORIAL3(IndicatorTestMediator, booleans, 2, booleans, 2, boolean
 /// @param p_gas          Control gas option
 /// @param p_steer        Control steering option
 /// @param p_force        Force feedback option
-void PControlTestMediator(bool p_steer, bool p_gas, bool p_brake, bool p_intervention, bool p_force)
+void PControlTestMediator(bool p_steer, bool p_gas, bool p_brake, bool p_intervention)
 {
     SDAConfigMediator::ClearInstance();
     ASSERT_TRUE(SetupSingletonsFolder());
-    tParticipantControl arr = {p_steer, p_gas, p_brake, p_intervention, p_force};
+    tParticipantControl arr = {p_steer, p_gas, p_brake, p_intervention};
```

Comment:
now p_force is unused, so remove it (from function, documentation and usages of function)